app/views/charges/create.html.erb

```diff
@@ -0,0 +1 @@
+<h2>Thanks, you paid <strong>$5.00</strong>!</h2>
```

Comment:
The $5.00 amount should probably be dynamic (as in, `<strong><%= @amount %></strong>` rather than hardcoded.